test/e2e/integreatly_test.go

```diff
@@ -106,6 +106,7 @@ func integreatlyWorkshopTest(t *testing.T, f *framework.Framework, ctx *framewor
 		"solution-explorer":     "tutorial-web-app-operator",
 		"user-sso":              "keycloak-operator",
 		"ups":                   "unifiedpush-operator",
+		"mdc":                   "mobile-developer-console-operator",
```

Comment:
Just move up list to keep list ordered alphabetically perhaps? Also, is it possible to add `mss` here as it got left out of this list in #122

```suggestion
		"mdc":                   "mobile-developer-console-operator",
		"mss":                   "mobile-security-service-operator",
```